AutowareAuto/src/perception/filters/point_cloud_fusion_nodes/launch/point_cloud_fusion.launch.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2024 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# This file is intended to launch point_cloud_fusion_nodes on CARMA freightliner trucks
+# consisting of two Velodyne LiDARs
+
+import launch
+from launch_ros.actions import Node
+from ament_index_python import get_package_share_directory
+
+import os
+
+point_cloud_fusion_node_pkg_prefix = get_package_share_directory('point_cloud_fusion_nodes')
+point_cloud_fusion_node_param_file = os.path.join(point_cloud_fusion_node_pkg_prefix,
+                                                  'param/point_cloud_fusion.param.yaml')
+
+
+def generate_launch_description():
+    point_cloud_fusion_nodes = Node(
```

Comment:
I think it could be beneficial from a performance perspective to launch both the `point_cloud_fusion_nodes` and `point_type_adapter` in a `ComposableNodeContainer`. If you can come up with an approach to achieve this from either drivers.launch.py or a separate launch file (maybe we launch both from point_cloud_fusion.launch.py?), I think it would be worth the effort. An example of how we launch `ComposableNodeContainer`s for CARMA Platform is located here: https://github.com/usdot-fhwa-stol/carma-platform/blob/3e7ca4fdb0e2256d7778cd98b9dfc496521ccfff/carma/launch/environment.launch.py#L361

Reply:
Just finished implementing this and testing on the truck